YAXLib/YAXSerializer.cs

```diff
@@ -1016,11 +1011,11 @@ private void AddNamespacesToElement(XElement rootNode)
         /// <param name="member">The member to serialize.</param>
         /// <param name="elementValue">The element value.</param>
         /// <param name="moveDescOnly">
-        ///     if set to <c>true</c> specifies that only the descendants of the resulting element should be
+        ///     if set to <see langword="true" /> specifies that only the descendants of the resulting element should be
         ///     added to the parent.
         /// </param>
         /// <param name="alreadyAdded">
-        ///     if set to <c>true</c> specifies the element returned is
+        ///     if set to <see langword="true" /> specifies the element returned is
```

Comment:
I curious, I personally never do this. But is there a real benefit when using this above `<c>true</c>`?

Reply:
I try to get used to it. With langword the xml help looks better to me, that's all.